vitrina/structure/views.py

```diff
@@ -3363,6 +3372,9 @@ def get_context_data(self, **kwargs):
         return context
 
     def form_valid(self, form):
+        if self.metadata_version.status != VersionStatus.DRAFT:
+            form.add_error(None, "Publikuota versija negali būti publikuota dar kartą.")
```

Comment:
Shouldn't this be translatable too?


```suggestion
            form.add_error(None, _("Publikuota versija negali būti publikuota dar kartą."))
```

Reply:
Yes, now it is, thanks